stanford-earth/SE3_BLT
# EARTH-273: Added new feature viewfield paragraph type to config.

# READY FOR REVIEW

# Summary
- New paragraph type feature - stanford-paragraph-viewfield

# Needed By (Date)
- July 13, 2017

# Steps to Test
1. git checkout —track origin/EARTH-273 
2. cd docroot/modules/stanford/stanford_paragraph_types
3. git checkout --track origin/EARTH-273
4. drush cim
5. drush csim local
6. drush features-import-all
7. go to local site in browser and go to: node/add/complex_page
8. Should see “Add Viewfield” in list of paragraph types
9. Add view field, set to view such as “recent content”, save and publish
10. resulting page should include view displayed using “filmstrip” layout

# Affected versions or modules
- N/A

# Associated Issues and/or People
- JIRA ticket EARTH-273
- PR https://github.com/SU-SWS/stanford_paragraph_types/pull/46
